.github/workflows/python-client-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Python Client Tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up JDK 17
+      uses: actions/setup-java@v4
+      with:
+        java-version: '17'
+        distribution: 'temurin'
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.9'
+
+    - name: Generate Python client library
+      run: build/sbt pythonClient/generate
+
+    - name: Install dependencies
+      run: |
+        python -m venv .venv
+        source .venv/bin/activate
+        pip install requests
+        pip install pytest
+    - name: Install unitycatalog Python package
```

Comment:
please use the script that you have added. if we dont use the script for actual CI testing, then how are we going to find out if that script accidentally breaks?